cmd/copy.go

```diff
@@ -242,7 +242,7 @@ func (raw rawCopyCmdArgs) cookWithId(jobId common.JobID) (cookedCopyCmdArgs, err
 	tempDest := raw.dst
 
 	if strings.EqualFold(tempDest, common.Dev_Null) && runtime.GOOS == "windows" {
-		tempDest = common.Dev_Null // map all capitalizations of "NUL"/"nul" to one because (on Windows) they all mean the same thing
+		tempDest = common.Dev_Null // map all capitalization of "NUL"/"nul" to one because (on Windows) they all mean the same thing
```

Comment:
"capitalizations" was correct here. On a side note, perhaps this is not the PR to run around correcting comment grammar in unrelated files... Though, we should certainly go through the code and ensure all comments make sense at some point.

Reply:
Adding to adreed's comment, changing the comments will alter the change description of the line. Change descriptions are helpful to know the history and reason for a change. I'd prefer if this file were unchanged.